### ISSUE TITLE
Hot Chocolate - Code First - Get Schema without running server

### ISSUE THREAD

**Issue by GeorgeTTD** [ChilliCream/graphql-platform#6078]
### Product

Hot Chocolate

### Is your feature request related to a problem?

We would like to generate a GraphQL schema to autogenerate downstream client libraries as part of our CI/CD pipelines. On larger projects we may have several dependencies which we may not to provision/stub/mock for the purpose of generating a schema file.

### The solution you'd like

It would be nice to have a way to generate the SDL file without having to physically run the server. I would liken this feature request to [this feature request for generating swagger docs for REST API](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/541). Their solution was to create a simple CLI which I believe uses reflection on the start-up file to generate the document. Their solution is [here](https://github.com/domaindrivendev/Swashbuckle.AspNetCore#retrieve-swagger-directly-from-a-startup-assembly).

I have checked and can't see an existing ticket for this requirement but happy to be corrected if its already been addressed. There may also be technical reasons why this wouldn't work.

**Comment by michaelstaib** [comment]
This is already implemented with 13.1 ... try the new preview with the new template:

```
dotnet new install HotChocolate.Templates::13.1.0-preview.7
dotnet new graphql
```

you can export the project for the new project like this

```
dotnet run -- schema export --output foo.graphql
```

you can also pipe things when leaving the output parameter away

**Comment by GeorgeTTD** [comment]
Amazing thankyou will give it a try.
